### ISSUE TITLE
feature: Simplify usage

### ISSUE THREAD

**Issue by keegancsmith** [sourcegraph/thyme#17]
Why don't we update Thyme CLI to have really smart defaults + "watch" built in. IE a user should just be able to run

```
$ thyme track
<ctrl-c>
$ thyme show
# opens thyme.html similiar to how go pprof opens webbrowser for svgs
```

and it would magically work.

**Comment by dufferzafar** [comment]
Going one step further, can't we just make all that transparent?

Just run `thyme` once, and it starts tracking in the background, dumping json etc. 

It also opens up a local webserver to serve the output html file, so to view the current results, just goto `localhost:6090` and they'll be served to you!

**Comment by HalfVoxel** [comment]
I don't think it is a good idea to start a web server without an explicit action from the user. But maybe a flag to be able to track and keep a server open at the same time would be a good idea.

Like
~~~
thyme --track --show
~~~

**Comment by seletskiy** [comment]
@dufferzafar: No, it's part of the service manager job to daemonize tools, not tool by itself. E.g., using systemd, there should be `.service` file managing that.

**Comment by beyang** [comment]
I like this idea. Will happily accept a PR :)
